contracts/BallotsStorage.sol

```diff
@@ -18,10 +18,10 @@ contract BallotsStorage is IBallotsStorage {
         _;
     }
 
-    function BallotsStorage(address _proxyStorage) public {
+    function BallotsStorage(address _proxyStorage, bool _demoMode) public {
```

Comment:
I don't agree on the param name. I think you can pass an integer instead
@varasev 

@vbaranov